teuthology/task/internal/syslog.py

```diff
@@ -33,6 +33,13 @@ def syslog(ctx, config):
         )
     )
 
+    # in systemd 237, the default is to forward to wall and syslog
```

Comment:
This is incorrect. On Ubuntu, the default is to forward to wall and **syslog** because Ubuntu developers patched systemd. It is not related to systemd version.